execution_engine_testing/test_support/src/wasm_test_builder.rs

```diff
@@ -941,6 +941,24 @@ where
         ret
     }
 
+    /// Gets all `[Key::Balance]`s in global state.
+    pub fn get_balances(&mut self) -> Vec<Key> {
```

Comment:
```suggestion
    pub fn get_balance_keys(&mut self) -> Vec<Key> {
```

Otherwise, `get_balances` makes it sound like it woud return _balances_ (`U512`) but this method actually returns keys.

Reply:
Applied suggestion